fusionauth/resource_fusionauth_tenant_helpers.go

```diff
@@ -262,6 +307,23 @@ func buildResourceDataFromTenant(t fusionauth.Tenant, data *schema.ResourceData)
 		return diag.Errorf("tenant.data: %s", err.Error())
 	}
 
+	connectorPolicies := []map[string]interface{}{}
+	// fusion auth always creates a default connector and connector policy. If you are not
+	// creating additional connectors, then reading in the default connector messes with
+	// the terraform state.
```

Comment:
If there is a better way to handle this, please let me know. 

Another option would be to always explicitly create a default connector even if there is no terraform config for it. That way it is in the terraform state and we don't need to ignore it here... but I'm not entirely sure how the terraform state is constructed, so I could be wrong about this 🤔

Reply:
Have you tried setting 'Computed' to true along with Optional?

Reply:
Ah finally figured this out... this is what the `Computed` attribute on the schema is for. I don't need to do anything fancy to handle this case, I'll remove my extra logic.

From the docs:
```
If Computed is true, then the result of this value is computed
(unless specified by config) on creation.
```

Reply:
lol, snap!!